src/mds/SessionMap.cc

```diff
@@ -608,6 +640,10 @@ void SessionMap::remove_session(Session *s)
 
   logger->set(l_mdssm_session_count, session_map.size());
   logger->inc(l_mdssm_session_remove);
+
+  utime_t avg_uptime;
+  avg_uptime.set_from_double(calc_avg_session_uptime(uptime.count()));
+  logger->tset(l_mdssm_avg_session_uptime, avg_uptime);
```

Comment:
I think this should just be a double, not a time.

Reply:
ACK. I'll raise a tracking ticket for this with the details required.